test/examples.civet

```diff
@@ -543,7 +543,7 @@ describe "examples (from real life)", ->
       debug: {
         module: serverModule,
       },
-    }
+    };
```

Comment:
We can get rid of this one by refining the rule:

```
( ... / ( !LexicalDeclaration ObjectLiteral ) ) InsertSemicolon
```

Reply:
That makes a lot of sense, but it breaks a lot of tests, presumably because of caching. I guess that's another danger of this hack, that `StatementDelimiter` now uses `Samedent` which might make it and all parents uncachable (with the current caching system). I'm not sure why the current hack doesn't break tests, but maybe best to leave it as is for now? Or feel free to help debug it. 😅